cohorts/cohort.py

```diff
@@ -1088,9 +1099,11 @@ def get_filtered_isovar_epitopes(self, epitopes, ic50_cutoff):
         return EpitopeCollection(mutant_binding_predictions)
 
     def load_single_patient_isovar(self, patient, variants, epitope_lengths):
-        # TODO: different epitope lengths, and other parameters, should result in
-        # different caches
-        isovar_cached_file_name = "%s-isovar.csv" % self.merge_type
+        # different cache depending on epitope length & variant set
+        variant_hash = make_hash(frozenset(variants))
```

Comment:
It does if 

1. you're on the same hardware/OS install
2. you put `export PYTHONHASHSEED=0` prior to running cohorts

Re the PYTHONHASHSEED, I set this in my `~/.bashrc`.  

One thing i've thought about doing is alerting the user if this env variable hasn't been set. Curious to know what you think - is this overkill?